aws/components/firewall/setup.ftl

```diff
@@ -49,13 +49,6 @@
         tags=getOccurrenceCoreTags(occurrence, core.FullName)
     /]
 
-                            {
-                                "Fn::Join": [
-                                    ",",
-                                    tierSubnetIdRefs
-                                ]
-                            },
-
```

Comment:
Not sure whats happened with this? might need the rebase?

Reply:
@roleyfoley, I removed it as noticed this unused code while was using the firewall component as a reference.

Reply:
Oh sorry its just floating in the middle of nowhere! Thanks for catching that